updating/installing-update-service.adoc

```diff
@@ -51,6 +51,12 @@ include::modules/update-service-create-service-web-console.adoc[leveloffset=+2]
 
 include::modules/update-service-create-service-cli.adoc[leveloffset=+2]
 
+[NOTE]
+====
+For the policy engine route if you see `ReconcileCompleted` status as `false`  with reason `CreateRouteFailed` caused by `host must conform to DNS 1123 naming convention`
+and `must be no more than 63 characters`, try creating the Update Service with a shorter name.
```

Comment:
> with a shorter name
I'd be more specific, e.g. `with a name that does not exceed 6 characters` or some such since they may have started with long name.

Reply:
> ... with a name that does not exceed 6 characters...

This assumes that their namespace is < 57 chars.  Do we still allow folks to pick their namespace, or is it hard-coded to `openshift-update-service` now?

Reply:
@jottofar I am not sure if we can be sure about the number of characters as the logic behind route host name can change in future.

Reply:
The name space needs to be `openshift-update-service `, thats the suggested namespace `operatorframework.io/suggested-namespace: openshift-update-service`

Reply:
They can select another namespace though.